Game/Source/ThirdPersonShooter/BigGym/BigGymGameMode.cpp

```diff
@@ -17,8 +20,142 @@ ABigGymGameMode::ABigGymGameMode()
 	if (PlayerPawnBPClass.Class != NULL)
 	{
 		DefaultPawnClass = PlayerPawnBPClass.Class;
+		PlayerCheckoutRadius = FGenericPlatformMath::Sqrt(PlayerPawnBPClass.Class.GetDefaultObject()->NetCullDistanceSquared);
 	}
-	
+
+	PlayersSpawned = 0;
+
 	// Seamless Travel is not currently supported in SpatialOS [UNR-897]
 	bUseSeamlessTravel = false;
 }
+
+void ABigGymGameMode::InitGameState()
+{
+	if (ShouldUseCustomSpawning())
```

Comment:
deliberately not calling super here?
If so add a comment why

Reply:
Nope, will add the super call